core/storage/email/gae_models.py

```diff
@@ -73,6 +73,19 @@ class SentEmailModel(base_models.BaseModel):
 
     @classmethod
     def _generate_id(cls, intent):
+    	"""Generates full message ID given the intent.
```

Comment:
This should be indented by 4 from the previous line, not 3.

Also, I'm not sure this is what this does? Perhaps "Generates an ID for a new SentEmailModel instance."